examples/user_guide/1_Plotting_Pitfalls.ipynb

```diff
@@ -400,7 +400,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Comparing **A&nbsp;** to **B&nbsp;**it should be clear that the \"fire\" colormap is revealing much more of the data, accurately rendering the density differences between each of the different blobs.  The unsuitable \"hot\" colormap is mapping all of the high density regions to perceptually indistinguishable shades of bright yellow/white, giving an \"oversaturated\" appearance even though we know the underlying heatmap array is *not* oversaturated (by construction).  Luckily it is easy to avoid this problem; just use one of the 50 perceptually uniform colormaps available in the [colorcet](https://github.com/bokeh/colorcet) package, one of the four shipped with matplotlib [(viridis, plasma, inferno, or magma)](https://bids.github.io/colormap), or the Parula colormap shipped with Matlab.\n",
+    "Comparing **A&nbsp;** to **B&nbsp;**it should be clear that the \"fire\" colormap is revealing much more of the data, accurately rendering the density differences between each of the different blobs.  The unsuitable \"hot\" colormap is mapping all of the high density regions to perceptually indistinguishable shades of bright yellow/white, giving an \"oversaturated\" appearance even though we know the underlying heatmap array is *not* oversaturated (by construction).  Luckily it is easy to avoid this problem; just use one of the 50 perceptually uniform colormaps available in the [colorcet](https://github.com/pyviz/colorcet) package, one of the four shipped with matplotlib [(viridis, plasma, inferno, or magma)](https://bids.github.io/colormap), or the Parula colormap shipped with Matlab.\n",
```

Comment:
```suggestion
    "Comparing **A&nbsp;** to **B&nbsp;**it should be clear that the \"fire\" colormap is revealing much more of the data, accurately rendering the density differences between each of the different blobs.  The unsuitable \"hot\" colormap is mapping all of the high density regions to perceptually indistinguishable shades of bright yellow/white, giving an \"oversaturated\" appearance even though we know the underlying heatmap array is *not* oversaturated (by construction).  Luckily it is easy to avoid this problem; just use a perceptually uniform colormap like one of the 60 available in the [colorcet](https://github.com/pyviz/colorcet) package, one of the four shipped with matplotlib [(viridis, plasma, inferno, or magma)](https://bids.github.io/colormap), or the Parula colormap shipped with Matlab.\n",
```